### ISSUE TITLE
Cannot get `send` to work

### ISSUE THREAD

**Issue by ctrlc03** [OpenZeppelin/nile#100]
I am having issues while trying to send a transaction via `send`. 

Steps

1.  `export PK1=123456789987654321`
2.  `nile  setup PK1 --network localhost`
3.  `nile send PK1 contract_alias test_function --network localhost`

Error:

```
Traceback (most recent call last):
  File "/usr/local/bin/nile", line 11, in <module>
    load_entry_point('cairo-nile===None', 'console_scripts', 'nile')()
  File "/usr/local/lib/python3.8/dist-packages/click-8.0.4-py3.8.egg/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/click-8.0.4-py3.8.egg/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/dist-packages/click-8.0.4-py3.8.egg/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/dist-packages/click-8.0.4-py3.8.egg/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/dist-packages/click-8.0.4-py3.8.egg/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/cairo_nile-None-py3.8.egg/nile/cli.py", line 100, in send
    account.send(contract_name, method, params)
  File "/usr/local/lib/python3.8/dist-packages/cairo_nile-None-py3.8.egg/nile/core/account.py", line 52, in send
    target_address, _ = next(deployments.load(to, self.network)) or to
StopIteration
```
Thank you

**Comment by martriay** [comment]
what version of nile and cairo contracts are you using?

**Comment by ctrlc03** [comment]
Sorry, I missed this.

Python: 3.8.10
Cairo: 0.8.0
Cairo contracts: 0.1.0
Nile: 0.5.2

First I deployed 2 account contracts with `nile setup PK1 --network localhost` and `nile setup PK2 --network localhost` 

Then I deployed the `contract_alias` with `nile deploy contract_alias arguments --network localhost` . Interaction with `invoke` and `call` works fine.

Finally I tried with `nile send PK1 contract_alias function_name --network localhost` 

The contract I have deployed is the mock `Ownable.sol` contract available in the `tests/mock/` directory of cairo-contracts.
